pkg/process/events/model/model_sysprobe_gen.go

```diff
@@ -55,6 +55,12 @@ func (z *ProcessMonitoringEvent) DecodeMsg(dc *msgp.Reader) (err error) {
 				err = msgp.WrapError(err, "CollectionTime")
 				return
 			}
+		case "exit_code":
```

Comment:
Is this file generated?

Reply:
yes it is, by https://github.com/DataDog/datadog-agent/blob/ff39eac67ff437cdfcb80d469472ef6f7952223d/pkg/process/events/model/model_sysprobe.go#L6

Reply:
Yes, it's generated using the `inv process-agent.go-generate` command.